### PR TITLE
feat: Implement first draft of writing the report to google docs.

### PR DESCRIPTION
**Pull request by iocanel**

### CODE REVIEW

**Comment by iocanel** [comment]
@cmoulliard it would be nice if you also gave a glimpse at this.

File: src/main/java/io/snowdrop/google/DocContentBuilder.java [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+        styleRequest = styleRequest.setFields(fields);
+      }
+      return new Request().setUpdateTextStyle(styleRequest);
+    }
+  }
+
+  private final int startIndex;
+  private int offset;
+  private int endIndex;
+  private final List<Entry> entries = new ArrayList<>();
```
Comment (geoand) [inline src/main/java/io/snowdrop/google/DocContentBuilder.java:19]: Oh come on, can't we have all these in the top? :P

File: src/main/java/io/snowdrop/google/GoogleDocsEndpoint.java [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+  @ConfigProperty(name = "google.docs.report.document-id")
+  String reportDocumentId;
+
+  @Inject
+  Docs docs;
+
+  @Inject
+  GithubReportingService reporting;
+
+  private void popullate() {
```
Comment (geoand) [inline src/main/java/io/snowdrop/google/GoogleDocsEndpoint.java]:
```suggestion
  private void populate() {
```

**Comment by geoand** [comment]
I would like to see this in action!

**Comment by iocanel** [comment]
> I would like to see this in action!

Shared the generated doc, just to get a glimpse. Still needs work, but its a rough idea of what we can do.

**Comment by geoand** [comment]
Great, thanks!

**Pull request merged**